models/page.go

```diff
@@ -138,6 +138,16 @@ func PostPage(p *Page) error {
 // Per the PUT Method RFC, it presumes all data for a page is provided.
 func PutPage(p *Page) error {
 	err := p.Validate()
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+
+	record, err := GetGroupByName(p.Name, p.UserId)
```

Comment:
As mentioned earlier, let's remove all the added checks in here, since it increases the scope of these changes. Also, I'm not quite sure what you're doing here, since you're getting a group when the user is editing a landing page?